cmd/collectors/rest/plugins/health/health.go

```diff
@@ -583,6 +625,26 @@ func (h *Health) getNodes() ([]gjson.Result, error) {
 	return result, nil
 }
 
+func (h *Health) getHADown() ([]gjson.Result, error) {
+	var (
+		result []gjson.Result
+		err    error
+	)
+
+	fields := []string{"possible,partner_name,state_description,partner_state"}
+	query := "api/private/cli/storage/failover"
+	href := rest.NewHrefBuilder().
+		APIPath(query).
+		Fields(fields).
+		Filter([]string{"possible=!true"}).
+		Build()
+
+	if result, err = collectors.InvokeRestCall(h.client, href, h.Logger); err != nil {
```

Comment:
might be clearer to change these lines to

`return collectors.InvokeRestCall(h.client, href, h.Logger)`